gammapy/data/observation_summary.py

```diff
@@ -125,3 +126,33 @@ def show_in_browser(self):
         """Make HTML file and images in tmp dir, open in browser.
         """
         raise NotImplementedError
+
+
+# class ObservationSummary(object):
```

Comment:
What's the plan with this class? Is it useful to keep this in a code comment here?
Maybe remove it and copy & paste it to somewhere else for yourself if you think it's useful?